nucypher/utilities/porter/control/specifications/fields/workorder.py

```diff
@@ -16,13 +16,13 @@
 """
 
 from nucypher.control.specifications.fields import Base64BytesRepresentation
-from nucypher.policy.orders import WorkOrder as WorkOrderClass
+from nucypher.policy.orders import ReencryptionRequest as ReencryptionRequestClass
 
 
-class WorkOrder(Base64BytesRepresentation):
-    def _serialize(self, value: WorkOrderClass, attr, obj, **kwargs):
-        return super()._serialize(value.payload(), attr, obj, **kwargs)
+class ReencryptionRequest(Base64BytesRepresentation):
+    def _serialize(self, value: ReencryptionRequestClass, attr, obj, **kwargs):
```

Comment:
Don't need to define a _serialize function if `bytes(value)` can be used for bytes representation since this is what the super class uses. WorkOrder needed to override the _serialize function because the `.payload()` function was needed for the corresponding bytes representation.